### PR TITLE
Remove unused EjectPage variable and make count size_t

### PR DESCRIPTION
**Pull request by AtariDreams**

### CODE REVIEW

File: filter/rastertoepson.c [hunk 1]
```diff
@@ -361,11 +360,9 @@ EndPage(
 
   free(Planes[0]);
 
-  if (CompBuffer)
-    free(CompBuffer);
+  free(CompBuffer);
```
Comment (michaelrsweet) [inline filter/rastertoepson.c:363]: Technically OK now, but all of the other CUPS code assumes C89 conformance which means `free(NULL)` is not safe...

File: filter/rastertoepson.c [hunk 2]
```diff
@@ -414,7 +411,7 @@ CompressData(const unsigned char *line,	/* I - Data to compress */
         		*start;		/* Start of compression sequence */
   unsigned char      	*comp_ptr,	/* Pointer into compression buffer */
 			temp;		/* Current byte */
-  int   	        count;		/* Count of bytes for output */
+  size_t   	    count;		/* Count of bytes for output */
```
Comment (michaelrsweet) [inline filter/rastertoepson.c:414]: `size_t` is overkill here (count always < 128) but no hard objection here.

File: filter/rastertoepson.c [hunk 3]
```diff
@@ -877,7 +874,7 @@ OutputRows(
 
     n = dot_count / DotBytes;
     putchar((int)(n & 255));
-    putchar((int)(n / 256));
+    putchar((int)(n >> 8));
```
Comment (michaelrsweet) [inline filter/rastertoepson.c:877]: Cosmetic since a good compiler should convert this to a shift anyways.

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Comment says removing Model, but you removed EjectPage.

Not sure about

**Comment by AtariDreams** [comment]
I made a mistake but it turns out that mistake works since EjectPage is also unused.

File: filter/rastertoepson.c (hunk 3 above)
Comment (AtariDreams) [reply filter/rastertoepson.c:876]: Yes but you use bitwise for the &255, so it should be clear that you putchar the low bits and then putchar the high bits

**Comment by AtariDreams** [comment]
@michaelrsweet addressed!

**Pull request closed without merging**